model/feature_flags.go

```diff
@@ -13,10 +15,35 @@ type FeatureFlags struct {
 
 	// Toggle on and off scheduled jobs for cloud user limit emails see MM-29999
 	CloudDelinquentEmailJobsEnabled bool
+
+	// Feature flags to control plugin versions
+	PluginIncidentManagement string `pluginid:"com.mattermost.plugin-incident-management"`
 }
 
 func (f *FeatureFlags) SetDefaults() {
 	f.TestFeature = "off"
 	f.TestBoolFeature = false
 	f.CloudDelinquentEmailJobsEnabled = false
+
+	f.PluginIncidentManagement = "1.1.1"
+}
+
+func (f *FeatureFlags) Plugins() map[string]string {
+	rFFVal := reflect.ValueOf(f).Elem()
+	rFFType := reflect.TypeOf(f).Elem()
+
+	pluginVersions := make(map[string]string)
+	for i := 0; i < rFFVal.NumField(); i++ {
+		rFieldVal := rFFVal.Field(i)
+		rFieldType := rFFType.Field(i)
+
+		pluginId, hasPluginId := rFieldType.Tag.Lookup("pluginid")
```

Comment:
super nit: `pluginid` vs `plugin_id` for increased ~grep~ ~ack~ ~ag~ ~rg~ rga goodness?

Reply:
Yup. Makes sense.

... rga?  googles... cool!